base/logger/DESCRIPTION

```diff
@@ -11,5 +11,5 @@ Suggests: testthat
 License: BSD_3_clause + file LICENSE
 Encoding: UTF-8
 LazyData: true
-RoxygenNote: 7.0.2
+RoxygenNote: 7.1.0
```

Comment:
Roxygen should be reverted to 7.0.2 here and in remote and emulator.  Things work better if we upgrade the entire system all at once, rather than having different PEcAn packages using different versions

Reply:
Sure, Thanks I will change it.